src/services/database/__tests__/registration-request-repository.integration.test.js

```diff
@@ -60,4 +63,37 @@ describe('Registration request repository', () => {
 
     expect(registrationRequest.status).toBe(status);
   });
+
+  describe('getNhsNumberByConversationId', () => {
+    it('should return the nhs number of a registration-request', async () => {
+      // given
+      const conversationId = uuid();
+      const odsCode = 'B12345';
+      const nhsNumber = '1234567890'
+      const status = Status.REGISTRATION_REQUEST_RECEIVED
+
+      await RegistrationRequest.create({
+        conversationId,
+        nhsNumber,
+        status,
+        odsCode
+      });
+
+      // when
+      const returnedNhsNumber = await getNhsNumberByConversationId(conversationId)
```

Comment:
Another small optional change, I would say the `await RegistrationRequest.create()` should be under the when.

```suggestion
      // when
      await RegistrationRequest.create({
        conversationId,
        nhsNumber,
        status,
        odsCode
      });
      
      const returnedNhsNumber = await getNhsNumberByConversationId(conversationId)
```